packages/eks-controller/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: eks-controller
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/system-charts/charts/eks-controller
+sources:
+  - "https://github.com/rancher/system-charts/charts/eks-controller"
+version: 0.1.0
```

Comment:
when does version match appversion? appversion is the tag for the image?

Reply:
appversion usually stands for the version of the underlying app. version stands for the version of helm chart.

Reply:
and when you say app, what does that mean? :) what does that encompass? what is an example of a diff between app and helm chart changes?

Reply:
@deniseschannon I think it usually refers to the image that is being deployed and supported. Helm docs isn't any more clear "appVersion: The version of the app that this contains (optional). This needn't be SemVer.": https://helm.sh/docs/topics/charts/

Reply:
@deniseschannon here is how it's explained in the docs: "Note that the appVersion field is not related to the version field. It is a way of specifying the version of the application. For example, the drupal chart may have an appVersion: 8.2.1, indicating that the version of Drupal included in the chart (by default) is 8.2.1. This field is informational, and has no impact on chart version calculations." https://helm.sh/docs/topics/charts/

Reply:
typically ends up being the version of the image that is being used and supported

Reply:
ok. i think this should be 0.1.000 to match with what we did with istio, where it's the appversion + `00` so you can increment when there are helm chart changes but the app version is the same.

Reply:
That Istio change was primarily so we wouldn't get out of line with upstream istio releases, but there is no upstream here. We could do it for uniformity but not necessary.

Reply:
agree that there is no need to add `00`